pkg/cloudprovider/providers/vsphere/vclib/diskmanagers/vdm.go

```diff
@@ -0,0 +1,72 @@
+package diskmanagers
+
+import (
+	"golang.org/x/net/context"
+
+	"github.com/golang/glog"
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/govmomi/vim25/types"
+	"k8s.io/kubernetes/pkg/cloudprovider/providers/vsphere/vclib"
+)
+
```

Comment:
There is vdm and vmdm. Maybe add one line description about them at the beginning to help ppl understand what's the difference.